src/components/Thread.vue

```diff
@@ -158,6 +172,14 @@ export default {
 		window.removeEventListener('resize', this.resizeDebounced)
 	},
 	methods: {
+		async updateSummary() {
+			if (this.thread.length < 2 && !this.enabledThreadSummary) return
+
+			this.summaryLoading = true
+			const summary = await summarizeThread(this.thread[0].databaseId)
+			this.summaryText = summary
+			this.summaryLoading = false
```

Comment:
This is triggered when navigating to another thread where the summary would be disabled. We should probably check the route to see if the user navigates away (to another thread).

Have a look at the following screen cast:

https://github.com/nextcloud/mail/assets/1479486/1fa7ff59-4604-4688-b0a9-5017e67b3627

Reply:
PS: This could also be related to my other comment above (regarding the `||` condition).